src/cs50/flask.py

```diff
@@ -22,7 +22,7 @@
 
     # Add support for Cloud9 proxy so that flask.redirect doesn't redirect from HTTPS to HTTP
     # http://stackoverflow.com/a/23504684/5156190
-    if getenv("C9_HOSTNAME") and not getenv("IDE_OFFLINE"):
+    if getenv("C9_PROJECT") == "ide50" and not getenv("IDE_OFFLINE"):
```

Comment:
How come better, @kzidane?

Reply:
`C9_HOSTNAME` is set offline as well.

Reply:
But that's why we're checking `and not getenv("IDE_OFFLINE")` too, no?

Reply:
Well, in that case we could just check for `IDE_OFFLINE` really since it's not set online. `C9_PROJECT` is set to a different value online (`"ide50"`) vs offline (`"ide50-offline"`).

Reply:
That'd be different logic. We need the ProxyFix iff using CS50 IDE Online.